hax/hax/util.py

```diff
@@ -281,6 +281,7 @@ def get_ha_fid(self) -> Fid:
         svc = self._local_service_by_name('hax')
         return mk_fid(ObjT.SERVICE, int(svc['ServiceID']) + 1)
 
+    @repeat_if_fails()
```

Comment:
Probably it is not that required now. get_rm_fid() is invoked in while preparing the entrypoint reply. If an exception (even HAConsistencyException) appears, EAGAIN will be returned to Motr.

`qconsumer` thread will not be blocked and perhaps will manage to do something meaningful instead of blocking on `@repeat_if_fails()`

Reply:
hmm.. yes that is true for entrypoint reply but get_rm_fid() is also invoked from motr::start() which may fail but this was true even earlier when we were fetching rm_fid from hax.py but _get_motr_fids() was tagged with `@repeat_if_fails()` in that case.
Have defined a nested function in motr::start() and tagged it with `repeat_if_fails()` and removed `repeat_if_fails()` from utils::get_rm_fid().

Reply:
Yeah, the more we use `@repeat_if_fails()`, the more tricky it is to understand whether repeat will happen or when it will not (especially when analyzing the long call hierarchies). At this point the idea of checked exceptions (Java) and explicit types like `IO a` (Haskell) look not that bad 😀